src/geo/map/torque-layer.js

```diff
@@ -90,6 +90,15 @@ var TorqueLayer = MapLayer.extend({
   },
 
   fetchAttributes: function(layer, featureID, callback) {
+  },
+
+  // given a timestamp returns a step (float)
+  timeToStep: function(timestamp) {
```

Comment:
Isn't this [overriding the timeToStep method on the L.TorqueLayer](https://github.com/CartoDB/torque/search?utf8=%E2%9C%93&q=timeToStep)? Is that on purpose?